client/app/components/Feed/Maker/Maker.jsx

```diff
@@ -1,12 +1,36 @@
-var React = require('react');
-
-var Maker = React.createClass({
-  render: function () {
-    return (
-      <div> Hello Super Sweet Maker
-      </div>
-    );
+import React from 'react';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+
+import RaisedButton from 'material-ui/RaisedButton';
+import TextField from 'material-ui/TextField';
+import Colors from 'material-ui/styles/colors.js';
+
+// console.log(Colors);
+// const styles = {
+//   errorStyle: {
+//     color: Colors.orange500,
+//   },
+//   underlineStyle: {
+//     borderColor: Colors.orange500,
+//   },
+// };
+
+let Maker = React.createClass({
```

Comment:
Why are we instantiating the class in this way? Is it in order to use the getInitialState function?

class ExampleComponent extends React.Component { constructor() { super(); this. _handleClick = this. _handleClick.bind(this); this.state = Store.getState(); }

It looks like maybe we can just store the state in this.state